src/content/docs/Build Your Project/build-commands.md

```diff
@@ -88,46 +90,47 @@ The built-in templates define two targets: `debug` (which is the default) and `r
 
 ## clean
 
-```bash
-clean
+```
+c3c clean
 ```
 
 ## build-run
 
-```bash
-build-run [target]
+```
+c3c build-run [target]
 ```
 
 Build the target (if needed) and run the executable.
 
 ## clean-run
 
-```bash
-clean-run [target]
+```
+c3c clean-run [target]
 ```
 
 Clean, build and run the target.
 
 ## dist
 
-```bash
-dist [target]
+```
+c3c dist [target]
 ```
 
-Clean, build and package the target.
+Clean, build and package the target for distribution.
 
-## docs
+## directives
 
-```bash
-docs [target]
 ```
+c3c directives [<target>]
+```
+                              
+Generate documentation for the target.
```

Comment:
No, it's docs. You generate documentation for the target. Note that this one is not complete yet and should be noted as such.